src/opm/parser/eclipse/EclipseState/Schedule/ActionAST.cpp

```diff
@@ -270,7 +270,7 @@ ASTNode ActionAST::parse_cmp(ActionParser& parser) {
 ASTNode ActionAST::parse_and(ActionParser& parser) {
     auto left = this->parse_cmp(parser);
     if (left.type == TokenType::error)
-        return TokenType::error;
```

Comment:
I kind of liked (ab)using the implicit conversion - but if that considered bad form we go down the `explicit` route.

Reply:
it's not abusing as such, it's always a trade off between potential errors and convenience. i'll drop.